templates/Containerisation/helm/helm-deploy.yaml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  - name: helmVersion
+    type: string
+    default: 'latest'
+  - name: azureSubscription
+    type: string
+    default: ""
+  - name: kubernetesCluster
+    type: string
+    default: "ss-dev-01"
+  - name: acrName
+    type: string
+    default: "sdshmctspublic"
+  - name: namespace
+    type: string
+    default: "vh"
+  - name: chartName
+    type: string
+  - name: chartPath
+    type: string
+  - name: fullChartName
+    type: string
+  - name: imageTag
+    type: string
+    default: "$(Build.BuildNumber)"
+
+steps:
+  - bash: |
+        ls -ltr ${{ parameters.chartPath }}/vh-bookings-api
+        echo "general filecheck"
+        ls -ltr . 
+        echo ${{ parameters.chartPath }}
+        echo ${{ parameters.fullChartName }}
+
+  - task: HelmDeploy@0
+    displayName: Helm upgrade
+    inputs:
+      connectionType: 'Azure Resource Manager'
+      azureSubscriptionEndpoint: '${{ parameters.azureSubscription }}'
+      azureResourceGroup: ${{ parameters.kubernetesCluster }}-rg 
+      kubernetesCluster: ${{ parameters.kubernetesCluster }}-aks
```

Comment:
Do we want to assume the format or let the user pass it in?

Reply:
this was me just being lazy and using an existing parameter. Better to let the user pass it in?

Reply:
I would as we would be making an assumption on the naming convention

Reply:
done and done